create.html

```diff
@@ -1,313 +1,344 @@
+        </main>
     </div>
-</div>
-
-<div class="fb-editorheader--form xgovuk-header__link govuk-header__link--service-name">Form Builder Console</div>
-<nav class="fb-editorheader--navigation">
-<ul>
-<li class="govuk-header__navigation-item"><a class="govuk-header__link" href="settings.html">Settings</a></li>
-</ul>
-</nav>
-</div>
-
-<nav class="fb-navigation">
-<ul>
-<li class="govuk-link"><a href="index.html">Forms</a></li>
-<li class="govuk-link"><a href="add.html">Add existing form</a></li>
-<li class="govuk-link fb-navigation-current"><a href="create.html">Create form</a></li>
-</ul>
-</nav>
-</header>
-    
-  <div class="govuk-width-container govuk-body-m">
-    
-
-
-    <main class="govuk-main-wrapper " id="main-content" role="main">
-      
-
-
-
-
-
-
-
-<div class="govuk-main-wrapper">
-<div class="govuk-grid-row">
-
-
-<div class="govuk-grid-column-two-thirds">
-
-<h1 class="govuk-heading-xl">Create form</h1>
-
-<div class="govuk-error-summary js-hidden" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary">
-  <h2 class="govuk-error-summary__title" id="error-summary-title">
-    There is a problem
-  </h2>
-  <div class="govuk-error-summary__body">
-    <ul class="govuk-list govuk-error-summary__list">
-      <li>
-        <a href="#createrepo" id="errorMessage"></a>
-      </li>
-    </ul>
-  </div>
-</div>
-
-
-<!-- CONTENTS -->
-    <div class="govuk-form-group govuk-form-group-target">
-      <label class="govuk-label" for="formname">
-        Form name
-      </label>
-      <span class="govuk-hint">
-        The name should contain only letters and numbers
-      </span>
-      <span id="input-error" class="govuk-error-message">
-      </span>
-      <input class="govuk-input govuk-input-target" id="formname" name="formname" type="text" pattern="^[a-z0-9]+$">
-    </div>
-    <div class="govuk-form-group createrepo-group">
-      <div class="govuk-checkboxes__item">
-        <input class="govuk-checkboxes__input" id="createrepo" name="createrepo" type="checkbox" value="yes">
-        <label class="govuk-label govuk-checkboxes__label" for="createrepo">
-          Create repository
-        </label>
-      </div>
-    </div>
-    <p id="gitInstructions">If you want to create a github repository for the form at the same time, please add your github details in <a href="settings.html">settings</a></p>
-
-    
-    <button type="submit" class="govuk-button" id="createForm">
-      Create form
-    </button>
-    
-<!-- CONTENTS END -->
-
-
-
-
-
-</div>
-
-</div>
-</div>
-
-    </main>
-  </div>
+    <script src="assets/govuk-frontend-design-system.min.js"></script>
+    <script>
+    const { execSync } = require('child_process')
+    const { app } = require('./app.js')
+    const { git } = app
+    const { pathExists, isDirectory, glob, request } = app.utils
+    const path = require('path')
+
+    function createErrorMessage(e, serviceName) {
+        let errorMessage
```

Comment:
is your convention to use 4 spaces? i think we use 2 everywhere else

Reply:
I added an HTML prettifier into Sublime, and this is how it does JS :(

I'll add the `eslint-plugin-html` and do it again :)

Reply:
I've left the HTML as the prettifier did it, but ESLint now has visibility of the JS in those files

Reply:
(The CLI tool "Prettifier" smooshes the JS, even with ESLint applied after. [FB-711](https://dsdmoj.atlassian.net/browse/FB-711) should take care of these things properly)